src/foam/nanos/notification/NotificationRowView.js

```diff
@@ -77,7 +78,7 @@
             ],
             obj: this.data
           })
-          .tag(this.NotificationCitationView, {
+          .tag(this.LiquidNotificationCitationView, {
```

Comment:
You can't reference this in FOAM .. it's in the NANOPAY code and doesn't make sense ...

Reply:
Ok, not sure which files should be moved in this case, should NotificationRowView be in nanopay?

Reply:
Wasn't exactly sure how this should be handled

Reply:
The default (FOAM) row view should just use the NotificationCitationView. There should be a NANOPAY version of the row view which that overrides the original and uses the LiquidNotificationCitationView.

Is there no way to dynamically lookup the proper ciatation view depending on the object being rendered . .. like Faceted view?

Reply:
You can to context.register(net.nanopay.liquid.LiquidNotificationCitationView, 'foam.nanos.notification.NotificationCitationView') to replace the view that gets used by default. Just like we do to replace the default ActionView with the nanopay ActionView.

Reply:
But also, I don't understand why Liquid would have a different notification view.